kuma/payments/jinja2/payments/terms.html

```diff
@@ -0,0 +1,57 @@
+{% extends 'base.html' %}
+
+{% block title %}{{ page_title(_('MDN Payment terms')) }}{% endblock %}
+
+{% block site_meta %}
+    {{ super() }}
+    {% set seo_description = _('MDN Payment terms') %}
+    {% set social_title = _('MDN Payment terms') %}
+    <meta property="og:description" content="{{ seo_description }}">
+    <meta name="description" content="{{ seo_description }}">
+    <meta name="twitter:description" content="{{ seo_description }}">
+    <meta name="twitter:title" content="{{ social_title }}">
+    <meta property="og:title" content="{{ social_title }}">
+{% endblock %}
+
+
+{% block content %}
+    <div>
+        <h2>Payment terms</h2>
+        <p>
+            Mozilla (that’s us) operates MDN Web Docs.
+        </p>
+        {% set url="https://www.mozilla.org/en-US/about/legal/terms/mozilla/" %}
+        <p>
+            These Payment Terms cover your payment to MDN. <a href="{{ url }}">The Websites & Communications Terms of Use</a>
+            cover the MDN service itself. If there is any inconsistency between these terms and the
+            Websites & Communications Terms of Use, these Payment Terms apply.
+        </p>
+        <h2>Payment Authorization</h2>
+        {% set terms_url="https://help.github.com/articles/github-terms-of-service/" %}
+        {% set privacy_policy_url="https://help.github.com/articles/github-privacy-statement/" %}
+        <p>
+            You can support MDN with a one-time payment, or you can choose to schedule monthly payments. Your payments are not tax deductible.
+            In order to sign up for monthly payments, you must sign in to MDN using a GitHub account.
+            GitHub has its own <a href="{{ terms_url }}">Terms</a> and <a href="{{ privacy_policy_url }}">Privacy Policy</a>.
+            When you choose to support MDN, you authorize us to charge the payment card you provide for the amount you choose, either as a one-time payment, or as a recurring monthly payment.
+        </p>
+        <h2>Cancellation</h2>
+        <p>
+            If you sign up for monthly payments, you may cancel at any time by sending an email to
+            <a href="mailto:mdn-support@mozilla.com">mdn-support@mozilla.com</a>.
+            If you choose to cancel, we will not charge your payment card for subsequent months.
+        </p>
+        <h2>Privacy Notice</h2>
+        {% set stripe_url="https://stripe.com/us/privacy" %}
+        {% set privacy_url="https://www.mozilla.org/en-US/privacy/websites/" %}
+        <p>
+            Your payment is processed by <a href="{{ stripe_url }}">Stripe</a>.
+        </p>
+        <p>
+            Mozilla receives a record of your payment and information about how much you’ve decided to
```

Comment:
In the source document, this says:

Mozilla receives a record of your payment, information about how much you’ve decided to pay, and whether you’ve set up monthly payments. Mozilla does not receive your payment details. If you sign up for an MDN account, Mozilla also receives the information you include in your account, as well as your email address from your Github logint. When Mozilla receives information from you, our [Mozilla Privacy Policy](https://www.mozilla.org/en-US/privacy/websites/) describes how we handle that information.

Maybe that bit about "your email address from your Github login" was a late addition? There some other changes, like an extra comma and some further prose. If so, this should reflect the source document, except that I think it should be spelled it "GitHub" (capital G and H) rather than "Githubt".

Reply:
Problem with this issue was that legal copy was amended after I have created PR :(
Now all should be up to date.